testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/assertions/DecisionInstanceAssert.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.process.test.api.assertions;
+
+import java.util.Map;
+
+/** The assertion object to verify a decision. */
+public interface DecisionInstanceAssert {
+  /**
+   * Verifies that the decision is evaluated. The verification fails if the decision is not
+   * evaluated or if the evaluation fails.
+   *
+   * <p>The assertion waits until the decision is evaluated.
+   *
+   * @return the assertion object
+   */
+  DecisionInstanceAssert isEvaluated();
+
+  /**
+   * Verifies that the decision has the given name.
+   *
+   * @param expectedDefinitionName the expected name for this decision
+   * @return the assertion object
+   */
+  DecisionInstanceAssert hasName(final String expectedDefinitionName);
+
+  /**
+   * Verifies that the definition has the given definition id.
+   *
+   * @param expectedDefinitionId the expected id for this decision
+   * @return the assertion object
+   */
+  DecisionInstanceAssert hasId(final String expectedDefinitionId);
+
+  /**
+   * Verifies that the definition has the given version.
+   *
+   * @param expectedVersion the expected version for this decision
+   * @return the assertion object
+   */
+  DecisionInstanceAssert hasVersion(final int expectedVersion);
```

Comment:
:wrench: I see no strong need to assert the decision name, the ID, or the version of the evaluated decision. The decision output is the interesting part.